### PR TITLE
Avoid concat of contact info in data resources import

### PR DESCRIPTION
**Pull request by vjrj**
A possible solution for #176.

### CODE REVIEW

File: grails-app/services/au/org/ala/collectory/EmlImportService.groovy [hunk 1]
```diff
@@ -20,18 +20,18 @@ class EmlImportService {
         guid:  { eml -> eml.@packageId.toString() },
         pubDescription: { eml -> this.collectParas(eml.dataset.abstract?.para) },
         name: { eml -> eml.dataset.title.toString() },
-        email: { eml ->  eml.dataset.contact?.electronicMailAddress?.text() },
+        email: { eml ->  eml.dataset.contact[0]?.electronicMailAddress?.text() },
```
Comment (djtfmartin) [inline grails-app/services/au/org/ala/collectory/EmlImportService.groovy:23]: @vjrj Im wondering what happens for datasets with zero contacts....

Comment (vjrj) [reply grails-app/services/au/org/ala/collectory/EmlImportService.groovy:23]: @djtfmartin I synced all our ipt DRs in empty testing collectory successfully but I need to check if we have some dr without contacts. Let me check.

Comment (vjrj) [reply grails-app/services/au/org/ala/collectory/EmlImportService.groovy:23]: @djtfmartin we were talking here and do some extra tests and the IPT does not allows to publish DRs without contacts.

Comment (vjrj) [reply grails-app/services/au/org/ala/collectory/EmlImportService.groovy:23]: For the record: https://eml.ecoinformatics.org/images/eml-dataset.png

2021-11-20-11-10-04-screenshot

Comment (djtfmartin) [reply grails-app/services/au/org/ala/collectory/EmlImportService.groovy:23]: Hmmm interesting. So its not a valid EML without a contact...

**Comment by vjrj** [comment]
Following here https://github.com/AtlasOfLivingAustralia/collectory/pull/100

**Pull request closed without merging**